<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>principale</class>
 <widget class="QMainWindow" name="principale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>principale</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Url :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="urlEdit">
       <property name="text">
        <string>https://api.waqi.info/feed/clermont-ferrand/?token=d9636de539c0cc32e3771e378fd51527b70526f1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>IQA :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="iqaSpinBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Niveau de pollution:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lvlEdit_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>461</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ANALYSE DU NIVEAU DE POLLUTION DE L'AIR (OU IQA) :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>502</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>urlEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>principale</receiver>
   <slot>changer_url()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>principale</receiver>
   <slot>iqa_afficher()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changer_url()</slot>
  <slot>iqa_afficher()</slot>
 </slots>
</ui>
